src/utils/fetch-course.ts

```diff
@@ -1,79 +1,27 @@
-import { fetchFile, fetchRaw } from '@/utils/fetch-github';
+import { CourseStructure } from '../lib/types';
+import { get } from 'fetch-unfucked';
+import { log, stringify } from '../utils/helpers';
 
-interface Result<T> {
-  data?: T;
-  error?: string;
-}
+const BRANCH: 'main' | 'draft' = 'draft';
```

Comment:
This is set to `draft` for now since `main` on soldev is waiting for this repo to be updated. Once we merge I'll send another PR to flip this to `main`. 

We could also use an env variable if we like.